.yarn/versions/12a089c0.yml

```
@@ -0,0 +1,2 @@
+releases:
+  "@yarnpkg/plugin-workspace-tools": minor
```

```suggestion
  "@yarnpkg/plugin-workspace-tools": patch
```
It following dev dependencies was a bug making this a patch, not a feature